static/app/components/smartSearchBar/index.tsx

```diff
@@ -1105,13 +1110,34 @@ class SmartSearchBar extends React.Component<Props, State> {
     this.setState(searchGroups);
   };
 
+  updateQuery = (newQuery: string, cursorPosition?: number) => {
```

Comment:
tiny nit, but you could drop the braces here since there's only one statement in this function body.